test/e2e/README.md

```diff
@@ -19,6 +19,7 @@ Use the selector engine [role-selector](https://playwright.dev/docs/selectors#ro
 ```js
 // Select a button with the accessible name "Hello World" (case-insensitive).
 page.locator( 'role=button[name="Hello World"i]' );
+page.getByRole( 'button', { name: 'Hello World' } ); // short-form
```

Comment:
TIL that the `name` filed in `getByRole` will automatically match the name case-insensitively. Might be good to briefly mention that here as well. WDYT?

Reply:
Updated in 23671315f8eb4b0b273615f21d9141f4e66e67ac.